src/NHSD.BuyingCatalogue.SolutionLists.Contracts/ISolutionOrganisation.cs

```diff
@@ -2,15 +2,15 @@
 
 namespace NHSD.BuyingCatalogue.SolutionLists.Contracts
 {
```

Comment:
File name is still ISolutionOrganisation, should be ISolutionSupplier

Reply:
Good spot. Changed